lib/model/rwfolder.go

```diff
@@ -1492,16 +1492,6 @@ func (p *rwFolder) inConflict(current, replacement protocol.Vector) bool {
 	return false
 }
 
-func invalidateFolder(cfg *config.Configuration, folderID string, err error) {
```

Comment:
This is only accidentally dead, as it should be used from a couple of places. There's an issue on fixing that in the planned or v0.13 milestones.